pkg/sql/logictest/testdata/logic_test/migrations

```diff
@@ -0,0 +1,51 @@
+# Confirm that crdb_internal.protect_table_for_session() lays a protected
```

Comment:
I'm a little out of the loop on sql logic test patterns, but this appears to have generated *several* calls to this test that run it in different configurations? Does this seem excessive? IIRC each of those will start a test server, wait for it to be up and running, then evaluate this file, then shut it down, which adds up over many tests/many configurations, so if the test isn't sensitive to the differences between configurations, I might see if we can avoid generating so many duplicates of this test?

Reply:
Good point.  I've restricted this to the `local` configuration, as I think we'll probably have adequate coverage by having this test run through a single configuration.